packages/reactive_ble_mobile/ios/Classes/ReactiveBlePlugin.m

```diff
@@ -1,5 +1,10 @@
 #import "ReactiveBlePlugin.h"
+
+#if __has_include(<reactive_ble_mobile/reactive_ble_mobile-Swift.h>)
 #import <reactive_ble_mobile/reactive_ble_mobile-Swift.h>
+#else
+#import "reactive_ble_mobile-Swift.h"
+#endif
```

Comment:
In which cases the original include doesn't work?

Reply:
Reason this is changed is to improve compatibility with host projects which do not use Swift or `use_frameworks!` (or use static frameworks - I can't remember).

There was a lengthy issue on the Flutter GH repo and all projects with native were migrated to this pattern.